pipeline/patch_VAE.py

```diff
@@ -173,14 +175,20 @@ def process_VAE(paths):
 
     assert len(set(s[:2] for s in sites)) == 1
     well = sites[0][:2]
+    print(f"\tloading file paths {os.path.join(temp_folder, '%s_file_paths.pkl' % well)}")
     fs = pickle.load(open(os.path.join(temp_folder, '%s_file_paths.pkl' % well), 'rb'))
+
+    print(f"\tloading static patches {os.path.join(temp_folder, '%s_adjusted_static_patches.pt' % well)}")
     dataset = torch.load(os.path.join(temp_folder, '%s_adjusted_static_patches.pt' % well))
     dataset = rescale(dataset)
     
     model = VQ_VAE(alpha=0.0005, gpu=True)
     model = model.cuda()
-    model.load_state_dict(torch.load('HiddenStateExtractor/save_0005_bkp4.pt'))
- 
+    if target:
+        model.load_state_dict(torch.load(target))
```

Comment:
I actually suggest using another keyword argument for model path (and also pickled pca path). Using `target` is fine, but it conflicts the input structure of other pipeline functions. I could help with the change!